migtests/tests/pg/partitions/fix-schema

```diff
@@ -10,3 +10,7 @@ sed -i  's/p2\.sydney/p2.sydney_region/g' $TEST_DIR/export-dir/schema/tables/tab
 sed -i  's/p2\.boston/p2.boston_region/g' $TEST_DIR/export-dir/schema/tables/INDEXES_table.sql
 sed -i  's/p2\.london/p2.london_region/g' $TEST_DIR/export-dir/schema/tables/INDEXES_table.sql
 sed -i  's/p2\.sydney/p2.sydney_region/g' $TEST_DIR/export-dir/schema/tables/INDEXES_table.sql
+
+sed -i  's/p2\.boston/p2.boston_region/g' $TEST_DIR/export-dir/assessment/reports/assessmentReport.json
```

Comment:
@sanyamsinghal pls add a comment in this file explaining the reason for the change in names (i.e. to test the scenario of manual change in partition names)

Reply:
done